todo_app/data/sessions.py

```diff
@@ -0,0 +1,69 @@
+from pymongo import MongoClient, UpdateOne
```

Comment:
Is anything using this file any more? Should we just delete it entirely?

Reply:
sessions.py is not in use. Has been removed.